src/screens/NewNote/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { API } from "aws-amplify";
+import React, { Component } from "react";
+import { ControlLabel, FormControl, FormGroup } from 'react-bootstrap';
+import LoaderButton from '../../components/LoaderButton';
+import config from '../../config';
+import { s3Upload } from '../../libs/awsLib';
+import './index.css';
+
+class NewNote extends Component {
+  constructor(props) {
+    super(props);
+
+    this.file = null;
```

Comment:
Any reason behind for declaring `file` in here?

Reply:
I have no idea @@
Like I said above and I don't even know why it is `class member`  instead of `state`
> just a clone from an existing component